### PR TITLE
chore: improve package.json

### LINKED ISSUE #62: Add `engines` field to `package.json`
Do we want to explain/warn when people are using a version of Node to develop that we don't support?

The lockfile is version 1, so I suppose we're only supporting Node 14 right now?

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Fill out `package.json` in a similar pattern as on the main Renovate repository

## Context:

Closes #62.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +13,35 @@
...
+  "keywords": [
+    "bitbucket",
+    "dependencies",
+    "dependency",
+    "docker",
+    "management",
+    "outdated",
+    "update"
+  ],
+  "author": "Maxime Brunet <insert-email-here>",
```
Comment (HonkingGoose) [inline package.json:29]: @maxbrunet Can you create a PR suggestion to make this the `author` field how you want it? I don't know how you want to be credited as the original author.

File: package.json [hunk 2]
```diff
@@ -11,6 +13,35 @@
...
+  "contributors": [
+    "HonkingGoose",
+    "Michael Kriese <michael.kriese@visualon.de>"
```
Comment (HonkingGoose) [inline package.json:32]: I've added myself and @viceice to the list of contributors.

File: package.json [hunk 3]
```diff
@@ -11,6 +13,35 @@
...
+  "bugs": {
+    "url": "https://github.com/renovatebot/renovate-approve-bot-bitbucket-cloud/issues"
+  },
+  "engines": {
+    "node": ">=14.15.0"
+  },
+  "engines-next": {
+    "description": "Versions other than the below are deprecated and a warning will be logged",
+    "node": ">=14.15.0"
+  },
```
Comment (HonkingGoose) [inline package.json:44]: Will need some help to set this up correctly. I've copy/pasted the `engines` and `engines-next` fields from the Renovate repository `package.json`.

Comment (viceice) [reply package.json:44]: engines-next can be removed

File: package.json [hunk 4]
```diff
@@ -11,6 +13,35 @@
...
+    "url": "https://github.com/renovatebot/renovate-approve-bot-bitbucket-cloud.git"
+  },
+  "keywords": [
+    "bitbucket",
+    "dependencies",
+    "dependency",
+    "docker",
+    "management",
+    "outdated",
+    "update"
```
Comment (maxbrunet) [inline package.json:27]: Would these keywords make more sense? (same as Github repo)
```suggestion
    "bitbucket",
    "bitbucket-cloud",
    "bot",
    "renovate"
```
Maybe `automerge` or `autoapprove` or `approve` or `approver` (or combinations with hyphens) could be good

Comment (HonkingGoose) [reply package.json:27]: I will add `automerge` myself.

**Review by viceice — APPROVED** [review]

**Review by maxbrunet — APPROVED** [review]

**Pull request merged**